File: javascript/retangulo.js
const base = parseFloat(prompt("Digite o valor da base"));
const altura = parseFloat(prompt("Digite o valor da altura"));

const area = base * altura;
const perimetro = 2 * (base + altura);
const diagonal = Math.sqrt(base * base + altura * altura);

console.log("Área = %.4f", area);
console.log("Perimetro = %.4f", perimetro);
console.log("Diagonal = %.4f", diagonal);

File: python/retangulo.py
from math import sqrt

base = float(input("Base do retangulo: "))
altura = float(input("Altura do retangulo: "))

area = base * altura
perimetro = base * 2 + altura * 2
diagonal = sqrt(base ** 2 + altura ** 2)

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")
